code/drasil-example/swhs/lib/Drasil/SWHS/GenDefs.hs

```diff
@@ -86,13 +86,15 @@ newtonLawNote u a c = foldlSent [ch u `S.is` S "found by assuming that",
 --  General Definitions Derivation  --
 --------------------------------------
 
-rocTempSimpDeriv :: Sentence -> [ConceptInstance] -> Derivation
-rocTempSimpDeriv s a = mkDerivName (S "simplified" +:+ phraseNP (rOfChng `of_` temp))
-  (weave [rocTempSimpDerivSent s a, map eS rocTempSimpDerivEqns])
+rocTempSimpDeriv :: (HasUID r, HasRefAddress r, HasShortName r) => r
```

Comment:
So this seems like a non-trivial change, and it looks due to `rocTempSimpDerivSent` changing, but I'm not clear why?

Reply:
Depending on how we view the above discussion, this can be reversed (in favour of creating at ticket to fix it later), document this a bit better, or we can do something else to improve it right now. 

The `r` parameter is to allow us to "pass the correct rocTempSimp" instance around without relying on both `rocTempSimp`s of NoPCM and SWHS happening to have the same UID.

Reply:
Yes to creating a ticket. We'll see how that goes.

The question to understand will be whether `rocTempSimp` is the same idea, and so should be shared as-is and have the same UID, or is actually different between NoPCM and SWHS, and so shouldn't be regarded as the same.

Reply:
Ok, I will reset this first in a commit (before creating the ticket) so that we have a single self-contained commit we can reference in a new ticket.